cmake/Modules/FindSoapySDR.cmake

```diff
@@ -9,7 +9,7 @@ if(NOT SOAPYSDR_FOUND)
     set(${VERSION} "${CMAKE_MATCH_1}" PARENT_SCOPE)
   endfunction(_SOAPY_SDR_GET_ABI_VERSION)
 
-  pkg_check_modules (LIBSOAPYSDR_PKG soapysdr>=0.4.0)
+  pkg_check_modules (LIBSOAPYSDR_PKG soapysdr>=0.4.0 SoapySDR>0.4.0)
```

Comment:
is there a reason for the greater than (>) on SoapySDR and the greater or equal (>=) on soapysdr. Shouldn't it be greater or equal for both?

Reply:
Nope, this was a type. I corrected it.